src/test/java/com/epam/tc/hw1/MultTest.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class MultTest extends AbstractTest {
+
+    @Test
+    public void multTest() {
+        double res = calculator.mult(3, 2);
+        assertThat(res).isEqualTo(6);
+    }
```

Comment:
Use dataproviders

_6.	All test data should be provided to test through the DataProviders_